aspnetcore/mvc/razor-pages/filter/sample/PageFilter/wwwroot/css/site.css

```diff
@@ -0,0 +1,37 @@
+/* Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification 
```

Comment:
Out of curiosity, do we always include all the css \ scripts with these samples?

Reply:
I lost that battle long ago. I used to exclude them with *.gitignore* and require you to manually override the ignore file. Now I ask folks to manually exclude them which is usually forgotten.